packages/components/libs/react-aria/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "main": "../../dist/cjs/reactAriaKZ.cjs",
   "module": "../../dist/esm/reactAriaKZ.mjs",
-  "types": "../../dist/types/__react-aria__/index.d.ts"
+  "types": "../../dist/types/__libs__/react-aria/index.d.ts"
```

Comment:
Small fix to correct the types for the libs export of RAC since I released this pointing to the incorrect path.